doc/release-process.md

```diff
@@ -29,6 +29,7 @@ get pushed to end-users.
 ## Let's get started
 
 To release a new version of Weave Gitops, you need to:
+- Verify that there are no outstanding PRs that need to be merge in the weave-gitops-dev channel on Slack, then notify the channel that you are starting the release process, and to not merge anything into main until you say otherwise.
```

Comment:
A tiny typo, `need to be merge` should be `need to be merged`.

❓  I am not good at English punctuation, it's hard, but is the comma required before `and` here? `and` here connects to items related to `notify` not the whole set/list of instructions.